<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CSkyDialog</class>
 <widget class="QDialog" name="CSkyDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>531</width>
    <height>338</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Sky</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QMenuBar" name="m_menuBar">
     <widget class="QMenu" name="m_menuGenerate">
      <property name="title">
       <string>&amp;Generate</string>
      </property>
      <addaction name="actionSkyCloudsMenu"/>
     </widget>
     <widget class="QMenu" name="m_menuSettings">
      <property name="title">
       <string>&amp;Settings</string>
      </property>
      <addaction name="actionSkyFogMenu"/>
     </widget>
     <addaction name="m_menuGenerate"/>
     <addaction name="m_menuSettings"/>
    </widget>
   </item>
   <item>
    <widget class="QToolBar" name="m_toolBar">
     <property name="iconSize">
      <size>
       <width>24</width>
       <height>24</height>
      </size>
     </property>
     <property name="floatable">
      <bool>false</bool>
     </property>
     <addaction name="actionSkyClouds"/>
     <addaction name="actionSkyFog"/>
     <addaction name="actionSkyWeather"/>
     <addaction name="separator"/>
     <addaction name="actionSkyNorth"/>
     <addaction name="actionSkyEast"/>
     <addaction name="actionSkySouth"/>
     <addaction name="actionSkyWest"/>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="title">
        <string>Cloud Preview</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="m_previewFrame">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>256</width>
            <height>256</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>256</width>
            <height>256</height>
           </size>
          </property>
          <property name="frameShape">
           <enum>QFrame::NoFrame</enum>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="ReflectedPropertyControl" name="m_propWnd" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="m_genStatus">
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="m_okButton">
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
  <action name="actionSkyCloudsMenu">
   <property name="text">
    <string>&amp;Clouds...</string>
   </property>
  </action>
  <action name="actionSkyFogMenu">
   <property name="text">
    <string>Fog...</string>
   </property>
  </action>
  <action name="actionSkyClouds">
   <property name="icon">
    <iconset>
     <normaloff>:/SkyDialog/bmp00007_0.png</normaloff>:/SkyDialog/bmp00007_0.png</iconset>
   </property>
   <property name="text">
    <string>Clouds</string>
   </property>
   <property name="toolTip">
    <string>Generate clouds</string>
   </property>
  </action>
  <action name="actionSkyFog">
   <property name="icon">
    <iconset>
     <normaloff>:/SkyDialog/bmp00007_1.png</normaloff>:/SkyDialog/bmp00007_1.png</iconset>
   </property>
   <property name="text">
    <string>Fog</string>
   </property>
   <property name="toolTip">
    <string>Fog</string>
   </property>
  </action>
  <action name="actionSkyWeather">
   <property name="icon">
    <iconset>
     <normaloff>:/SkyDialog/bmp00007_2.png</normaloff>:/SkyDialog/bmp00007_2.png</iconset>
   </property>
   <property name="text">
    <string>Weather</string>
   </property>
   <property name="toolTip">
    <string>Modify weather settings</string>
   </property>
  </action>
  <actiongroup name="directionActionGroup">
   <action name="actionSkyNorth">
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>:/SkyDialog/bmp00007_3.png</normaloff>:/SkyDialog/bmp00007_3.png</iconset>
    </property>
    <property name="text">
     <string>North</string>
    </property>
    <property name="toolTip">
     <string>Change preview direction to North</string>
    </property>
   </action>
   <action name="actionSkyEast">
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>:/SkyDialog/bmp00007_4.png</normaloff>:/SkyDialog/bmp00007_4.png</iconset>
    </property>
    <property name="text">
     <string>East</string>
    </property>
    <property name="toolTip">
     <string>Change preview direction to East</string>
    </property>
   </action>
   <action name="actionSkySouth">
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>:/SkyDialog/bmp00007_5.png</normaloff>:/SkyDialog/bmp00007_5.png</iconset>
    </property>
    <property name="text">
     <string>South</string>
    </property>
    <property name="toolTip">
     <string>Change preview direction to South</string>
    </property>
   </action>
   <action name="actionSkyWest">
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>:/SkyDialog/bmp00007_6.png</normaloff>:/SkyDialog/bmp00007_6.png</iconset>
    </property>
    <property name="text">
     <string>West</string>
    </property>
    <property name="toolTip">
     <string>Change preview direction to West</string>
    </property>
   </action>
  </actiongroup>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ReflectedPropertyControl</class>
   <extends>QWidget</extends>
   <header>Controls/ReflectedPropertyControl/ReflectedPropertyCtrl.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
